scripts/upgrade-users-sparse-index.js

```diff
@@ -0,0 +1,58 @@
+﻿'use strict';
+// Set the Node ENV
+process.env.NODE_ENV = 'development';
+
+var chalk = require('chalk'),
+  mongoose = require('../config/lib/mongoose');
+
+mongoose.loadModels();
+
+var _indexToRemove = 'email_1',
+  errors = [],
+  processedCount = 0;
+
+mongoose.connect(function (db) {
+  // get a reference to the User collection
+  var userCollection = db.connections[0].collections.users;
+
+  console.log(chalk.yellow('*************** Removing index "' + _indexToRemove + '" from the User collection ***************'));
```

Comment:
@ilanbiala Yes. I don't really like what I did with the messages. I'll clean them up.

Just to clarify, are you concerned with the length of the line of the code, or the output of the console log?